### PR TITLE
mlx - add mx.round and fix numpy args

### PR DESCRIPTION
**Pull request by lkarthee**
Align `mlx/numpy.py` with other implementations. There were some missing params which are added.

### CODE REVIEW

File: keras/src/backend/mlx/numpy.py [hunk 1]
```diff
@@ -158,12 +158,12 @@ def arctanh(x):
     return mx.arctanh(x)
 
 
-def argmax(x, axis=None):
+def argmax(x, axis=None, keepdims=False):
     x = convert_to_tensor(x)
     return mx.argmax(x, axis=axis)
```
Comment (fchollet) [inline keras/src/backend/mlx/numpy.py:163]: We should also add support for the `keepdims` arg here

File: keras/src/backend/mlx/numpy.py [hunk 2]
```diff
@@ -158,12 +158,12 @@ def arctanh(x):
...
 
 
-def argmin(x, axis=None):
+def argmin(x, axis=None, keepdims=False):
     return mx.argmin(x, axis=axis)
```
Comment (fchollet) [inline keras/src/backend/mlx/numpy.py:167]: Same here

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras/src/backend/mlx/numpy.py [hunk 3]
```diff
@@ -158,13 +158,13 @@ def arctanh(x):
...
+def argmax(x, axis=None, keepdims=False):
     x = convert_to_tensor(x)
-    return mx.argmax(x, axis=axis)
+    return mx.argmax(x, axis=axis, keepdims=keepdims)
 
 
-def argmin(x, axis=None):
-    return mx.argmin(x, axis=axis)
+def argmin(x, axis=None, keepdims=False):
+    return mx.argmin(x, axis=axis, keepdims=keepdims)
```
Comment (lkarthee) [inline keras/src/backend/mlx/numpy.py:167]: do we need to add convert_to_tensor to all funcs? Few have it and others don't.

Comment (fchollet) [reply keras/src/backend/mlx/numpy.py:167]: Generally, yes, we should always use it on tensor arguments.

**Review by fchollet — APPROVED** [review]

**Pull request merged**